packages/core/src/codewhisperer/client/codewhisperer.ts

```diff
@@ -227,18 +227,18 @@ export class DefaultCodeWhispererClient {
         const client = await this.createUserSdkClient()
         const requester = async (request: CodeWhispererUserClient.ListAvailableCustomizationsRequest) =>
             client.listAvailableCustomizations(request).promise()
-        return pageableToCollection(requester, {}, 'nextToken')
+        return pageableToCollection(requester, {}, 'nextToken' as never)
             .promise()
             .then((resps) => {
                 let logStr = 'amazonq: listAvailableCustomizations API request:'
-                resps.forEach((resp) => {
+                for (const resp of resps) {
                     const requestId = resp.$response.requestId
                     logStr += `\n${indent('RequestID: ', 4)}${requestId},\n${indent('Customizations:', 4)}`
-                    resp.customizations.forEach((c, index) => {
+                    for (const [c, index] of enumerate(resp.customizations)) {
```

Comment:
Does using enumerate() mean we are iterating the collection twice? The `index` might be a good argument for using `forEach` in this case.